[5.11.0] - Remove unnecessary references and promote stored procedures (#2891)
wso2/docs-is: en/docs/setup/removing-unused-tokens-from-the-database.md

You can use one of the following methods for token cleanup.

!!! note

>     We recommend using stored procedures instead of using the cleanup task that is available in the WSO2 Identity 

Reviewer: ```suggestion
    We recommend using stored procedures instead of using the cleanup task that is available in WSO2 Identity 
```

You can use one of the following methods for token cleanup.

!!! note

>     We recommend using stored procedures instead of the WSO2 Identity Server to cleanup tokens since there 

Reviewer: Could we explain what we mean by `cleaning up tokens using WSO2 Identity Server`?

Author: Fixed with https://github.com/wso2/docs-is/pull/2891/commits/94dc4de619234388f1e95f1bbd8a03ec7ab875ea.

    have been a few reported issues when the WSO2 Identity server is used for token cleanup.

### Using stored procedures for token cleanup

You can use the provided stored procedures to run a
token cleanup task periodically to remove the old and invalid tokens.
Follow the instructions below to configure token cleanup using this
method.

!!! tip

>     It is safe to run these steps in read-only mode or during a time when traffic on the server is low, but that is not mandatory.

Reviewer: ```suggestion
    It is safe to run these steps in read-only mode or during a time when traffic on the server is low but that is not mandatory.
```

> 1. Disable the internal token cleanup process by configuring the following property in the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder.

Reviewer: ```suggestion
1. Disable the internal token cleanup process by configuring the following property in the `deployment.toml` file (which is stored in the `<IS_HOME>/repository/conf` folder).
```

      3. [MySQL](https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/mysql/token-cleanup/)
      4. [Oracle](https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/oracle/token-cleanup/)
      5. [PostgreSQL 9.X](https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-9x/token-cleanup/)
      6. [PostgreSQL 11.X](https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-11x/token-cleanup/)

3. Once the cleanup is over, start the WSO2 Identity Server pointing to
   the cleaned-up database. You can also schedule a cleanup task that will be automatically run
   after a given period of time.

### Configuring WSO2 Identity Server for token cleanup

> Alternatively, you can also use the WSO2 Identity Server which triggers token cleanup during the following

Reviewer: ```suggestion
Alternatively, you can also use the WSO2 Identity Server, which triggers token cleanup during the following
```
